tyr/servers/server.py

```diff
@@ -21,6 +21,10 @@ class Server(object):
     NAME_SEARCH_PREFIX = '{envcl}-{location}-'
     NAME_AUTO_INDEX = True
 
+    GLOBAL_IAM_ROLE_POLICIES = ['allow-get-chef-artifacts-chef-client',
```

Comment:
Could you format the list with entries and brackets on separate lines? [Like this][0].

[0]: https://github.com/hudl/Tyr/blob/master/tyr/servers/solr/slave.py#L9